src/topology/category/Top/limits.lean

```diff
@@ -101,3 +102,96 @@ instance forget_preserves_colimits : preserves_colimits (forget : Top.{u} ⥤ Ty
       (colimit_cocone_is_colimit F) (types.colimit_cocone_is_colimit (F ⋙ forget)) } }
 
 end Top
+
+namespace Top
+
+section topological_konig
+
+/-!
+## Topological Kőnig's lemma
+
+A topological version of Kőnig's lemma is that the inverse limit of nonempty compact Hausdorff
+spaces is nonempty.  (Note: this can be generalized further to inverse limits of nonempty compact
+T0 spaces, where all the maps are closed maps; see A.H. Stone, "Inverse limits of compact spaces").
```

Comment:
Could you add this to `docs/references.bib`? (Sorry, no good deed goes unpunished.)

Reply:
No worries -- and I found through MathSciNet that there was an erratum for that theorem (which might partially explain why I wasn't able to follow some of the steps!)